drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -1130,93 +1142,58 @@
 
 /*** Assessment window notification ***/
 .assessment-notification {
```

Comment:
You can reuse the instructions styling here for the popup, since they are mostly the same.

Reply:
Done